lib/views/public_body/_more_info.html.erb

```diff
@@ -62,6 +62,13 @@
   <% end %>
 <% end %>
 
+<% if public_body.has_tag?('cqc') %>
```

Comment:
Left a query on https://github.com/mysociety/whatdotheyknow-theme/issues/1364#issuecomment-1235318944 regarding this - it _**may**_ make sense to split into `cqcprovider` and `cqclocation` (as many have both, which may not be the same thing!).